beets/autotag/mb.py

```diff
@@ -218,10 +219,12 @@ def track_info(recording, index=None, medium=None, medium_index=None,
                     lyricist.append(artist_relation['artist']['name'])
                 elif type == 'composer':
                     composer.append(artist_relation['artist']['name'])
+                    composer_sort.append(artist_relation['artist']['sort-name'])
```

Comment:
This line is now too long, which is why our [linting test](https://travis-ci.org/beetbox/beets/jobs/227173206) is failing. You'll need to wrap it to be under 80 characters, the easiest way being this:

```Python
composer_sort.append(
    artist_relation['artist']['sort-name'])
```